guest/internal/cyclestate/cyclestate.go

```diff
@@ -7,15 +7,3 @@
 //
 // See https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/#extension-points
 package cyclestate
-
-import (
-	"sigs.k8s.io/kube-scheduler-wasm-extension/guest/api"
-	"sigs.k8s.io/kube-scheduler-wasm-extension/guest/api/proto"
-	"sigs.k8s.io/kube-scheduler-wasm-extension/guest/internal/prefilter"
-)
-
-// Pod is the current pod being scheduled. It is lazy and the same values are
-// returned for any plugins in a scheduling cycle.
-var Pod proto.Pod = prefilter.Pod
-
-var Values api.CycleState = prefilter.CycleState
```

Comment:
Why did you remove this? I might forget the context.

Reply:
@Gekko0114 
I removed this for fixing type error of removing `predfilterextensions` and merging `_addPod/_removePod`.
I was testing this change in my local environment and this affects a lot and I will resume this.